### DISCUSSION TITLE
import lume failed (Netlify deploy)

### DISCUSSION THREAD

**Discussion by Isaac-Tait** [lumeland/lume#129] [Q&A] [answered]
I am getting this error message: 
```
Error: Import 'https://deno.land/x/lume@v0.25.3/deps/yaml.js' failed: 404 Not Found
8:18:24 PM:     at file:///opt/build/repo/admin/config.tmpl.js#1626146304012:1:0
```
My `config.tmpl.js` file can be viewed [here](https://github.com/Isaac-Tait/macadamGrinding_denoStyle/blob/main/admin/config.tmpl.js).

I am not certain how to go about clearing the error so Netlify can build/deploy my site. Any pointers would be appreciated. Cheers!

**Comment by oscarotero** [comment, marked as answer]
Hi.
Lume is using typescript now, so you have to update your imported files from `*.js` to `*.ts`.
In this file it should be:

```ts
import { stringify } from "lume/deps/yaml.ts";
```

`_config.js` is still working for now with `*.js` but I encourage to change to `*.ts`, to prevent future errors when javascript files are removed.
Thanks!

**Comment by Isaac-Tait** [comment]
Thank you @oscarotero I made that small fix and the error is gone and the Netlify deploy was built. Cheers!
